### PR TITLE
New Pages

### PR DESCRIPTION
**Pull request by Subtain-Farooq**
A new support Page, Blog list, and blog post page were added. Some issues are fixed on all pages.

### CODE REVIEW

File: content/about/index.de.md [hunk 1]
```diff
@@ -11,82 +11,4 @@ section_engagement_title: 'Unser Engagement'
 section_customers_title_prefix: 'Unsere'
 section_customers_title: 'Kunden & Partner'
 
-engagements:
```
Comment (hendrikebbers) [inline content/about/index.de.md:14]: this looks wrong. The current main branch do not contain that code anymore. Why is it removed here? @Subtain-Farooq can you please merge the main branch in your branch and try if that helps. There are several places like this in the PR.

Comment (Subtain-Farooq) [reply content/about/index.de.md:14]: Okay, I will update it.

Comment (hendrikebbers) [reply content/about/index.de.md:14]: THX. If you have any issue please ping me. In that I can try to fix it.

Comment (Subtain-Farooq) [reply content/about/index.de.md:14]: Could you please delete this PR? Resolving this issue taking time so I have created a new branch [new-pages](https://github.com/OpenElements/website/tree/new-pages) and this branch is updated with the master branch.

**Pull request closed without merging**